src/frontend/mame/ui/selgame.cpp

```diff
@@ -1355,8 +1355,21 @@ void menu_select_game::inkey_export()
 	}
 	else
 	{
+		if (isfavorite())
+		{
+			// iterate over favorites
+			for (auto & favmap : mame_machine_manager::instance()->favorite().m_list)
+			{
+				if (favmap.second.startempty == 1)
+					list.push_back(favmap.second.driver);
+				else
+					return;
+			}
+		}
+		else
```

Comment:
Please indent the other line that's been made the `else` branch of this new `if` statement.  Apart from that this change is good.